Resources/Audio/_Harmony/Animals/attributions.yml

```
@@ -0,0 +1,4 @@
+- files: ["gorilla_scream.ogg"]
+  license: "CC-BY-SA-3.0"
+  copyright: "Taken from Paradise Station modified by Ermucat for SS14"
+  source: "https://github.com/ParadiseSS13/Paradise/blob/master/sound/creatures/gorilla.ogg"
```

Attributions should point to a specific commit, or it may become invalid over time
```suggestion
  source: "https://github.com/ParadiseSS13/Paradise/blob/bdb8039c8fda42407af70443d5d50de2e9ec72c6/sound/creatures/gorilla.ogg"
```